src/main/java/com/hrznstudio/titanium/material/ResourceRegistry.java

```diff
@@ -41,14 +40,15 @@
 
     public static AdvancedTitaniumTab RESOURCES = new AdvancedTitaniumTab("resources", true);
     public static Logger LOGGER = LogManager.getLogger(Titanium.MODID + "-ResourceRegistry");
+    public static final String PLUGIN_NAME = "resources";
 
     private static HashMap<String, ResourceMaterial> MATERIALS = new HashMap<>();
     private static HashMap<String, HashMultimap<String, Field>> ANNOTATED_FIELDS = new HashMap<>();
     private static Field modifiersField;
 
     public static void onInit() {
         scanForReferences();
-        ModLoader.get().postEvent(new ResourceRegistrationEvent());
+        Titanium.RESOURCES.execute(PluginPhase.CONSTRUCTION);
```

Comment:
wouldn't it just be easier to have the main mod return a List of PluginHandlers, and have it handle the phases over directly having to call the individual ones?